const testUncaughtExceptionController = () => {
    throw new Error('Simulated uncaught exception');
};

export default testUncaughtExceptionController;
